### PR TITLE
Pypi

### PR DESCRIPTION
**Pull request by stdavis**
This PR makes this package publishable to pypi and consequently pip-installable.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -65,9 +78,14 @@ A normalized string representing the entire address that was passed into the con
...
+    - `pip install -e ".[develop]"`
 1. install required dependencies to run sweeper tests
-   - `pip install -e ".[test]"`
+    - `pip install -e ".[test]"`
 1. run tests: `pytest`
+
+### Uploading to pypi.org
+
+1. `python setup.py sdist`
+1. `twine upload dist/*`
```
Comment (steveoh) [inline readme.md]: what is twine?

Comment (stdavis) [reply readme.md]: The recommended way of uploading to pypi now. If you try to do the old way `python setup.py upload` it fails and points you to twine.

Comment (steveoh) [reply readme.md]: is twine something i would pip install or what is that?

Comment (steveoh) [reply readme.md]: the twine package shows `python setup.py sdist bdist_wheel`. Did you try creating a wheel? I remember reading they are better than the original...

Comment (stdavis) [reply readme.md]: I see that in the docs now. It wasn't in the tutorial I was following. I built a wheel and uploaded it as well for v1.0.3 so now it looks like this:
image

I'll update the readme...

**Pull request merged**